src/core/manager.c

```diff
@@ -3537,3 +3551,11 @@ static const char *const manager_state_table[_MANAGER_STATE_MAX] = {
 };
 
 DEFINE_STRING_TABLE_LOOKUP(manager_state, ManagerState);
+
+static const char *const ctrl_alt_del_burst_action_table[_BURST_ACTION_MAX] = {
+        [BURST_ACTION_IGNORE] = "ignore",
+        [BURST_ACTION_REBOOT] = "reboot",
+        [BURST_ACTION_POWEROFF] = "poweroff",
```

Comment:
Those should be called "reboot-force" and "poweroff-force", they correspond to `systemtl reboot --force`, `systemtl poweroff --force`. Othewise, it'd be too easy to mistake with a normal reboot.